frontend/src/components/contracts/ContractDetails.tsx

```diff
@@ -122,13 +120,15 @@ export default class extends React.Component<Props, State> {
             <Col md="4">
               <CardCell
                 title={
-                  <Term title={"Locked?"}>
-                    <p>{`Locked contract means that there are no access keys allowing the
+                  <Term
+                    title={"Locked?"}
+                    text={`
+                      Locked contract means that there are no access keys allowing the
                       contract code to be re-deployed (e.g. even a single FullAccess
-                      permission access key casts the locked status to "No").`}</p>
-                    <p>{`Note: The contract itself may be implemented to re-deploy itself
-                      or re-add FullAccess keys.`}</p>
-                  </Term>
+                      permission access key casts the locked status to "No"). \n
```

Comment:
We lost the paragraphs here

![image](https://user-images.githubusercontent.com/304265/108758462-e0c44480-7529-11eb-8396-73252326c15a.png)